src/translations.php

```diff
@@ -219,6 +219,15 @@
         "Longest Week Streak" => "Найбільша к-сть тижнів",
         "Present" => "Наразі",
     ],
+    "ur_PK" => [
```

Comment:
This should preferably be just "ur", unless Urdu (Pakistan) is different enough from Urdu (India) that it needs to be translated separately.

```suggestion
    "ur" => [
```

Reply:
Hi. So there are a few structural differences between the two versions and there are some words as well that are either spelt differently or are totally different. I think it would be better that they are kept different, but later on if someone comes in to translate for "ur" I think they will be able to alias this if they believe ur_PK works for that version too.